openssl-dynamic/src/main/c/sslsession.c

```diff
@@ -124,4 +124,6 @@ jint netty_internal_tcnative_SSLSession_JNI_OnLoad(JNIEnv* env, const char* pack
     return TCN_JNI_VERSION;
 }
 
-void netty_internal_tcnative_SSLSession_JNI_OnUnLoad(JNIEnv* env) { }
+void netty_internal_tcnative_SSLSession_JNI_OnUnLoad(JNIEnv* env, const char* packagePrefix) {
+    netty_internal_tcnative_util_unregister_natives(env,packagePrefix, "io/netty/internal/tcnative/SSLSession");
```

Comment:
consider making a `const` for `"io/netty/internal/tcnative/SSLSession"` like in other modules.

Reply:
ups... I missed this one